common/persistence/wrappers/sampled/visibility_manager.go

```diff
@@ -44,21 +42,18 @@ const (
 // errPersistenceLimitExceededForList is the error indicating QPS limit reached for list visibility.
 var errPersistenceLimitExceededForList = &types.ServiceBusyError{Message: "Persistence Max QPS Reached for List Operations."}
 
-type visibilitySamplingClient struct {
-	rateLimitersForOpen   *domainToBucketMap
-	rateLimitersForClosed *domainToBucketMap
-	rateLimitersForList   *domainToBucketMap
-	persistence           VisibilityManager
-	config                *SamplingConfig
+type sampledVisibilityManager struct {
```

Comment:
nit: already moved this file to `sampled` package so the prefix is redundant and can be dropped from the name. `sampled.sampledVisibilityManager` -> `sampled.visibilityManager`